### ISSUE TITLE
Error from `predict.mmblogit()`

### ISSUE THREAD

**Issue by fweber144** [melff/mclogit#23]
With mclogit v0.9.4, the following example runs into an error:
```r
data(VA, package = "MASS")
ngrp <- 8L
VA$grp <- gl(n = ngrp,
             k = floor(nrow(VA) / ngrp),
             length = nrow(VA),
             labels = paste0("gr", seq_len(ngrp)))
set.seed(2643)
VA$grp <- sample(VA$grp)

fit_mblogit <- mclogit::mblogit(cell ~ treat + age + Karn,
                                random = ~ 1 | grp,
                                data = VA)

lpreds <- predict(fit_mblogit, newdata = VA)

```
The last line throws
```
Error in `[.data.frame`(mf, group.labels) : undefined columns selected
```
which I guess is due to the changes in `predict.mmblogit()` from commit 371692d4998970a60988255d3796de58c91c7797.

**Comment by melff** [comment]
Murphy's Law in action, one bug fixed, the other bug created. 
But this bug appears to be fixed in release 0.9.4.1

**Comment by fweber144** [comment]
Yeah, I know that :D Thanks for fixing.
